embedding/cpp/flutter_service_app.cc

```diff
@@ -98,3 +98,34 @@ bool FlutterServiceApp::OnCreate() {
 
   return true;
 }
+
+void FlutterServiceApp::ParseEngineArgs() {
+  char *app_id;
+  if (app_get_id(&app_id) != 0) {
+    TizenLog::Warn("App id is not found.");
+    return;
+  }
+  std::string temp_path("/home/owner/share/tmp/sdk_tools/" +
```

Comment:
Maybe that path will change later, so how about subtracting it with a variable or define? And I think a method other than hard coding would be better...

Reply:
@bwikbs https://github.com/flutter-tizen/flutter-tizen/pull/19 explains why we use this dirty hack.

The path should never change because it's the only directory sdb has a read/write access to on TV. If you're to change the hard-coded value to a constant, pls make sure to make a consistent change in the C# embedding as well.

Reply:
@swift-kim Thanks for letting me know! 
What do you think of the aul parameter way? Shall we give it a try?

Reply:
Uhm, #19 explains why we can't use other way than the current way. We used to use \_\_DLP_UNIT_TEST_ARG__ to pass arguments to launched apps before the PR but that method didn't work for TV devices.

Reply:
Ah, I didn't think of commercial TV. Thanks for letting me know.